flink-runtime/src/main/java/org/apache/flink/runtime/dispatcher/DefaultJobManagerRunnerFactory.java

```diff
@@ -70,7 +70,9 @@ public JobManagerRunner createJobManagerRunner(
                     slotPoolServiceSchedulerFactory.getSchedulerType()
                             == JobManagerOptions.SchedulerType.Adaptive,
                     "Adaptive Scheduler is required for reactive mode");
-            ReactiveModeUtils.configureJobGraphForReactiveMode(jobGraph);
+            JobGraphConfigurationUtils.configureJobGraphForReactiveMode(jobGraph);
+        } else {
+            JobGraphConfigurationUtils.configureJobGraphForDefaultMode(jobGraph);
```

Comment:
I think it makes sense to still keep the logic together in the `JobGraphConfigurationUtils` instead of two separate util classes – do you agree there?

Reply:
Yes!